src/Sfa.Roatp.Indexer.Core/Models/RoatpProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Sfa.Roatp.Indexer.Core.Models
+{
+    public class RoatpProvider
+    {
+        public string Ukprn { get; set; }
+
+        public ProviderType ProviderType { get; set; }
+
+        public bool ContractedForNonLeviedEmployers { get; set; }
+
+        public bool ParentCompanyGuarantee { get; set; }
+
+        public bool NewOrganisationWithoutFinancialTrackRecord { get; set; }
+
+        public DateTime StartDate { get; set; }
+
+        public DateTime EndDate { get; set; }
+
+        public bool IsEqual(RoatpProviderDocument oldRoatpProvider)
```

Comment:
Maybe put this method in a separate class to determine if the RoatpProvider from the spreadsheet is already in the index or the same.